docs/tutorials/launch-jupyter-lab-tutorial.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 2
```

Comment:
Let's put it as first. The Stable Diffusion tutorial is too convoluted, so makes sense to have simple tutorials first.